test/integration/load_node_test.cpp

```diff
@@ -78,6 +78,36 @@ TEST(LoadNodeTest, IterateMap) {
   EXPECT_EQ(3, i);
 }
 
+TEST(LoadNodeTest, AliasAccess) {
+  const Node doc = Load("{A: &DEFAULT {str: string, int: 42, float: 3.1415}, B: *DEFAULT}");
+  const Node& A = doc["A"];
+  const Node& B = doc["B"];
+
+  // A and B have the same content
+  ASSERT_TRUE(A);
+  ASSERT_TRUE(A.IsMap());
+  ASSERT_TRUE(B);
+  ASSERT_TRUE(B.IsMap());
+
+  // A and B have the same content
```

Comment:
I think A and B are actually the same node, so you can just check

ASSERT_TRUE(A.is(B));

or

ASSERT_TRUE(A == B);

(which is the same thing).